samples/tools/autogenbench/pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "autogenbench"
 authors = [
-  { name="Autogen Team", email="auto-gen@outlook.com" },
```

Comment:
Why change this? Does auto-gen@outlook.com not work anymore?

Reply:
Please see other comments up on the thread.